bin/database_consistency

```diff
@@ -1,9 +1,32 @@
 #!/usr/bin/env ruby
 
-base_dir = File.join(Dir.pwd, ARGV.first.to_s)
-
-unless File.realpath(base_dir).start_with?(Dir.pwd)
-  puts "\nWarning! You are going out of current directory, ruby version may be wrong and some gems may be missing.\n"
+if ARGV.include?('install')
+  config_filename = '.database_consistency.yml'
+  rules = <<RULES
+# Ignore false positive from Rails' ActionText and ActiveStorage
+ActionText::RichText:
```

Comment:
Let's put this to somefile because later we can have more default rules.

Reply:
I moved the rules to `lib/database_consistency/templates/rails_defaults.yml`.
Please change or let me know if you prefer another location.

Reply:
I think it's fine.